release/models/aft/openconfig-aft-common.yang

```diff
@@ -385,6 +407,37 @@ submodule openconfig-aft-common {
     }
   }
 
+  grouping aft-common-entry-nexthop-gre-state {
+    description
+      "GRE encapsulation applied on a next-hop";
+
+    leaf src-ip {
+      type union {
+        type oc-inet:ip-address;
+        type string;
+      }
+      description
```

Comment:
Given that this is state, can we not just simplify this down to only using the IP address? The implementation can resolve what address is being used if needs be.

Reply:
Makes sense. I will make it just IP address.